docs/application/tizen-studio/native-tools/batt-historian.md

```diff
@@ -0,0 +1,157 @@
+
+Battery Historian tool helps you to increase the productivity of Tizen devices by maximizing the performance with efficient use of resources. Battery Historian tool enables you to analyze and audit the behavior and consumption of battery in the Tizen devices. It is an expert analysis tool that precisely reviews information and events related to the battery. It includes various metrics and insights, such as radio cells, communication, and so on. Battery Historian tool generates a report that helps you to diagnose and troubleshoot the device behavior related to battery consumption. The generated report also helps you to identify precise battery-draining elements so that you can fine-tune and augment Tizen device productivity and efficiency.
+
+Battery Historian tool also allows an A/B comparison of two bug reports, highlighting the key differences in the battery-related metrics.
+
+## Prerequisites
+
+To use the Battery Historian feature, ensure that: 
+
+ - The latest version of Tizen Studio is installed on your development hardware.
+ - The Smart Development Bridge (SDB) is enabled to issue commands to Tizen devices. 
+
+## Access Battery Historian
+
+To access and open the Battery Historian tool, use one of the following:
+
+ - In the Tizen Studio menu, select **Tools \> Battery Historian.**
+ - In the Tizen Studio toolbar, click **Battery Historian**  ![](./media/batt1.png).
+ - In Tizen Studio main window, press the combination of the **Alt+Shift+B** keys. 
+
+The **Battery Historian** window appears. 
+
+![](./media/bug1.png)
+
+The Battery Historian tool window comprises of the following components:
+
+ - **Browse**: This component allows you to browse and select a bug report file.
+ - **Switch to Bug Report Comparison**: This component allows you to compare and analyze two different bug report files.
+
+**Figure** : Battery Historian bug report comparison window
+
+![](./media/bug2.png)
+
+> [!NOTE]
+> The Battery Historian tool only supports **.txt** or **.zip** format for bug report file.
+
+## Use Battery Historian Tool
+
+The Battery Historian tool essentially requires a bug report file to process the battery consumption stats of a Tizen device. This bug report file contains all the relevant data regarding the battery consumption. You can generate the bug report using the SDB that dumps the battery data to your development machine and creates a report.
+
+The Battery Historian tool takes the bug report file as an input to analyze and compare the stats. After processing the report data, the Battery Historian uses HTML format to visualize the stats in a web browser.
+
+To use the Battery Historian tool, ensure that you have the bug report files readily available. If you do not have the bug report file with, you can generate one using the following procedure:
```

Comment:
First developer have to create abnormal power scenario before generating the bug report, should we clearly mention it?

Reply:
@sijupunnnoose
Yes, that can be one of the use cases too to portray the comparison with visible differences. 

The following are the primary use cases:
- To get insights about the components that are utilizing more power in one device Tweaking the components and then comparing the previous stats with the latest stats to check whether there is an improvement or not.
- To get patterns of power consumption so as to make the device more intelligent in power saving. 

We did not have the battery dumps that could have portrayed the use case which you have mentioned. So, kindly let me know what will be the best use case to proceed with. 

Thanks,